src/components/welcome/MyAccounts/index.tsx

```diff
@@ -59,60 +76,89 @@ const AccountsList = ({ safes, onLinkClick }: AccountsListProps) => {
                 <Button
                   disableElevation
+                  variant="outlined"
                   size="small"
                   onClick={onLinkClick}
                   startIcon={<SvgIcon component={AddIcon} inheritViewBox fontSize="small" />}
+                  sx={{ height: '36px', width: '100%', px: 2 }}
                 >
                   Add
                 </Button>
               </Link>
             </Track>
-          }
-          noSafesMessage={NO_WATCHED_MESSAGE}
-          onLinkClick={onLinkClick}
-        />
+            <Track {...OVERVIEW_EVENTS.CREATE_NEW_SAFE} label={trackingLabel}>
+              <CreateButton isPrimary={!!wallet} />
+            </Track>
+          </Box>
+        </Box>
+
+        <Paper className={css.safeList}>
+          {/* Pinned Accounts */}
+          <Box mb={2} minHeight="170px">
+            <div className={css.listHeader}>
+              <SvgIcon
+                component={BookmarkIcon}
+                inheritViewBox
+                fontSize="small"
+                sx={{ mt: '2px', mr: 1, strokeWidth: 2 }}
+              />
+              <Typography variant="h5" fontWeight={700} mb={2}>
+                Pinned
+              </Typography>
+            </div>
+            {pinnedSafes.length > 0 ? (
+              <SafesList safes={pinnedSafes} onLinkClick={onLinkClick} />
+            ) : (
+              <Box className={css.noPinnedSafesMessage}>
+                <Typography color="text.secondary" maxWidth="350px" textAlign="center" fontSize="14px">
+                  Personalize your account list by clicking the
+                  <SvgIcon
+                    component={BookmarkIcon}
+                    inheritViewBox
+                    fontSize="small"
+                    sx={{ mx: '4px', color: 'text.secondary', position: 'relative', top: '2px' }}
+                  />
+                  icon on the accounts most important to you.
+                </Typography>
+              </Box>
+            )}
+          </Box>
 
+          {/* All Accounts */}
+          <Accordion sx={{ border: 'none' }}>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon sx={{ '& path': { fill: 'var(--color-text-secondary)' } }} />}
+              sx={{ padding: 0, '& .MuiAccordionSummary-content': { margin: '0 !important', mb: 1, flexGrow: 0 } }}
+            >
+              <div className={css.listHeader}>
+                <Typography variant="h5" fontWeight={700}>
+                  Accounts
+                  {allSafes && allSafes.length > 0 && (
+                    <Typography component="span" color="text.secondary" fontSize="inherit" fontWeight="normal" mr={1}>
+                      {' '}
+                      ({allSafes.length})
+                    </Typography>
+                  )}
+                </Typography>
+              </div>
+            </AccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
```

Comment:
As an idea should we keep the accounts list open by default if there are no pinned safes?

Reply:
good idea!